tests/test_load_data.py

```diff
@@ -195,7 +222,7 @@ def test_grid_wrf(self):
         result = can_i_run_smrf(config)
         self.assertTrue(result)
 
-    @unittest.skip(" Skipping over hrrr tests for issues with eccodes")
+    #@unittest.skip(" Skipping over hrrr tests for issues with eccodes")
```

Comment:
Are these needed? I'd suggest removing to not confuse down the road.

Reply:
Not needed. Just laziness on my part. Removing.